sample/SampleAspNetCoreApp/Startup.cs

```diff
@@ -14,15 +14,20 @@ public class Startup
 	{
 		public Startup(IConfiguration configuration) => Configuration = configuration;
 
-		public IConfiguration Configuration { get; }
-
-		// This method gets called by the runtime. Use this method to add services to the container.
+		public IConfiguration Configuration { get; }
+
+		// This method gets called by the runtime. Use this method to add services to the container.
 		public void ConfigureServices(IServiceCollection services)
 		{
 			ConfigureServicesExceptMvc(services);
 
-			services.AddMvc()
-				.SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+#if NETCOREAPP3_0
+			services.AddRazorPages().SetCompatibilityVersion(CompatibilityVersion.Version_3_0);
+#elif NETCOREAPP2_2
+			services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+#else

```

Comment:
First of all, I don't see any reasons to add an addition target for `netcoreapp3.1` due to no breaking changes introduced with .Net Core 3.1 release. In case `netcoreapp2.0` and `netcoreapp2.1` was a different story.

Regarding, your comment I definitely agree with you and will open new PR with fixing that.

Reply:
Regarding `netcoreapp3.1`: Since 3.1 is the LTS version, I think that is the one we should test - that version will be with us for a very long time - so I'd either replace 3.0, or add 3.1 additionally. I just think that from this point adding an additional target framework is not a lot of work and having 1 more would not hurt. To me this seems like a very low hanging fruit - with some additional lines of change we can cover both 3.0 and 3.1.

Reply:
> Regarding `netcoreapp3.1`: Since 3.1 is the LTS version, I think that is the one we should test - that version will be with us for a very long time - so I'd either replace 3.0, or add 3.1 additionally. I just think that from this point adding an additional target framework is not a lot of work and having 1 more would not hurt. To me this seems like a very low hanging fruit - with some additional lines of change we can cover both 3.0 and 3.1.

I got your point after #645. I thought from `Elastic.Apm` perspective. Never mind.